feature/eks-cluster-creation/aws-eks.tf

```diff
@@ -1,15 +1,14 @@
 module "eks" {
   source          = "terraform-aws-modules/eks/aws"
   version         = "12.2.0"
-  cluster_name    = var.eks_cluster_name
-  subnets         = module.vpc.private_subnets
+  cluster_name    = var.cluster_name
+  subnets         = module.vpc.public_subnets
   vpc_id          = module.vpc.vpc_id
   cluster_version = var.eks_cluster_version
-
+  enable_irsa     = true
```

Comment:
can you describe this value? I don't know what it does. Some comment here would be helpful.

Reply:
Just did it. This value is required to enable IAM Roles for Service Accounts which are created during terraform run.